Server/src/BaseQuasarServer.cpp

```diff
@@ -335,27 +339,77 @@ int BaseQuasarServer::parseCommandLine(
     if (vm.count("help"))
     {
         cout << desc << endl;
-        if (isHelpOrVersion)
-            *isHelpOrVersion = true;
+        if (isHelpOrVersionOrExtra)
+            *isHelpOrVersionOrExtra = true;
         return 0;
     }
+
     //Print version if needed
     if (printVersion)
     {
-    	std::cout << VERSION_STR << std::endl << \
-    			"\t BuildHost: " << BuildMetaInfo::getBuildHost() << std::endl << \
-				"\t BuildTimestamp: " << BuildMetaInfo::getBuildTime() << std::endl << \
-				"\t CommitID: " << BuildMetaInfo::getCommitID() << std::endl << \
-				"\t ToolkitLibs: " << BuildMetaInfo::getToolkitLibs() << std::endl;
-        if (isHelpOrVersion)
-            *isHelpOrVersion = true;
+        #ifdef BACKEND_UATOOLKIT
+        std::string uasdkCoreModuleVersionInfo = VersionInfoCoreModule::getCoreModuleVersionInfo().toUtf8();
+        #else
+        std::string uasdkCoreModuleVersionInfo = "N/A";
```

Comment:
Suggest to say "N/A in open62541 backend" or so to be more clear why the information is not there.